packer/scripts/create-kube-certs.sh

```diff
@@ -2,6 +2,8 @@
 
 set -o errexit -o xtrace
 
+export PATH="${PATH}:/usr/local/bin/"
```

Comment:
Mainly, because it's a global solution to a local problem, and making that change feels _wrong_.